### PR TITLE
🐛 Fix missing cnquery dependencies and docs

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

File: policy/policy.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.7
+// 	protoc        v3.21.5
```
Comment (chris-rock) [inline policy/policy.pb.go]: We should remove this change, you are using an old protoc compiler

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+cnquery
```
Comment (chris-rock) [inline .gitignore:1]: Should this be `./cnquery`?

Comment (czunker) [reply .gitignore:1]: For me, only this or `/cnquery` works.

File: policy/policy.pb.go (hunk 1 above)
Comment (czunker) [reply policy/policy.pb.go]: Fixed.

**Pull request merged**